tests/CMakeLists.txt

```diff
@@ -74,6 +74,34 @@ foreach(file ${base_psa_generated_data_files})
 endforeach()
 
 if(GEN_FILES)
+    add_custom_command(
+        OUTPUT
+            ${CMAKE_CURRENT_SOURCE_DIR}/src/test_keys.h
+        WORKING_DIRECTORY
+            ${CMAKE_CURRENT_SOURCE_DIR}
+        COMMAND
+            ${MBEDTLS_PYTHON_EXECUTABLE}
+                "${CMAKE_CURRENT_SOURCE_DIR}/scripts/generate_test_keys.py"
+                    "--output"
+                        "${CMAKE_CURRENT_SOURCE_DIR}/src/test_keys.h"
```

Comment:
Pre-existing and minor, but I find it weird to increase the indentation level for every line. 
```
        COMMAND
            ${MBEDTLS_PYTHON_EXECUTABLE}
            "${CMAKE_CURRENT_SOURCE_DIR}/scripts/generate_test_keys.py"
            "--output"
            "${CMAKE_CURRENT_SOURCE_DIR}/src/test_keys.h"
        DEPENDS
```
would make more sense to me. (Conceptually it's a flat list that's going to be passed to `exec()` or equivalent.)